Catappult/appcoins-wallet-android
# [feature/APPC-2781] Onboarding 2.0

**What does this PR do?**

   New onboarding layout

**Database changed?**

 No

**Where should the reviewer start?**

- OnboardingFragment.kt

**How should this be manually tested?**

- open wallet and test the recover wallet
- clean data, test the bottom sheet to check the links redirect and the wallet creation

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-2781

**Code Review Checklist**
